app/portainer/views/auth/authController.js

```diff
@@ -261,6 +263,9 @@ class AuthenticationController {
         this.LocalStorage.cleanLogoutReason();
       }
 
+      // always clear the kubernetes cache on login
+      cache.store.clear();
```

Comment:
It could be worth calling dispatchCacheRefreshEvent() - this makes sure to clear the angular and react caches